changelog/1453.feature.rst

```diff
@@ -0,0 +1,10 @@
+Created `plasmapy.formulary.misc` to contain functionality for
+miscellaneous plasma parameters, and migrated
+`~plasmapy.formulary.misc._grab_charge`,
```

Comment:
```suggestion
``~plasmapy.formulary.misc._grab_charge``,
```
...since it's not part of the public API.  An additional possibility would be to include `_grab_charge` in a `trivial` changelog entry since it's not part of the public API, but I have no problem with keeping it here.

Reply:
I debated this myself.  I think just adding the double back-ticks here will suffice.